datasets/covid19_google_mobility/mobility_report/pipeline.yaml

```diff
@@ -0,0 +1,161 @@
+
+dag:
+  airflow_version: 1
+  initialize:
+    dag_id: mobility_report
+    default_args:
+      owner: "Google"
+
+      depends_on_past: False
+      start_date: "2021-03-01"
+    max_active_runs: 1
+    schedule_interval: "@daily"
+    catchup: False
+    default_view: graph
+
+  tasks:
+    - operator: "KubernetesPodOperator"
+
+      description: "Run CSV transform within kubernetes pod"
+
+      args:
+
+        task_id: "mobility_report_transform_csv"
+
+        startup_timeout_seconds: 600
+        name: "mobility_report"
+
+        namespace: "default"
+
+        affinity:
+          nodeAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              nodeSelectorTerms:
+                - matchExpressions:
+                    - key: cloud.google.com/gke-nodepool
+                      operator: In
+                      values:
+                        - "pool-e2-standard-4"
+
+
+        image_pull_policy: "Always"
+
+        image: "{{ var.json.covid19_google_mobility.container_registry.run_csv_transform_kub }}"
+
+        env_vars:
+          SOURCE_URL: "https://www.gstatic.com/covid19/mobility/Global_Mobility_Report.csv"
+          SOURCE_FILE: "files/data.csv"
+          TARGET_FILE: "files/data_output.csv"
+          TARGET_GCS_BUCKET: "{{ var.value.composer_bucket }}"
+          TARGET_GCS_PATH: "data/covid19_google_mobility/mobility_report/data_output.csv"
+          PIPELINE_NAME: "mobility_report"
+          CSV_HEADERS: >-
+            ["country_region_code" ,"country_region" ,"sub_region_1" ,"sub_region_2" ,"metro_area" ,"iso_3166_2_code" ,"census_fips_code" ,"place_id" ,"date" ,"retail_and_recreation_percent_change_from_baseline" ,"grocery_and_pharmacy_percent_change_from_baseline" ,"parks_percent_change_from_baseline" ,"transit_stations_percent_change_from_baseline" ,"workplaces_percent_change_from_baseline" ,"residential_percent_change_from_baseline"]
+          RENAME_MAPPINGS: >-
+            {"country_region_code":"country_region_code" ,"country_region":"country_region" ,"sub_region_1":"sub_region_1" ,"sub_region_2":"sub_region_2" ,"metro_area":"metro_area" ,"iso_3166_2_code":"iso_3166_2_code" ,"census_fips_code":"census_fips_code" ,"place_id":"place_id" ,"date":"date" ,"retail_and_recreation_percent_change_from_baseline":"retail_and_recreation_percent_change_from_baseline" ,"grocery_and_pharmacy_percent_change_from_baseline":"grocery_and_pharmacy_percent_change_from_baseline" ,"parks_percent_change_from_baseline":"parks_percent_change_from_baseline" ,"transit_stations_percent_change_from_baseline":"transit_stations_percent_change_from_baseline" ,"workplaces_percent_change_from_baseline":"workplaces_percent_change_from_baseline" ,"residential_percent_change_from_baseline":"residential_percent_change_from_baseline"}
```

Comment:
Use the `json` package as follows: https://github.com/GoogleCloudPlatform/public-datasets-pipelines/pull/174#discussion_r717929113

Reply:
As discussed we are going with this as of now... 
Hence, not changing the code...